src/Cake.Npm/NpmUpdateAliases.cs

```diff
@@ -0,0 +1,71 @@
+namespace Cake.Npm
+{
+    using System;
+    using Cake.Npm.Update;
+    using Core;
+    using Core.Annotations;
+
+    /// <summary>
+    /// Npm Update aliases
+    /// </summary>
+    [CakeAliasCategory("Npm")]
+    [CakeNamespaceImport("Cake.Npm.Update")]
+    public static class NpmUpdateAliases
+    {
+        /// <summary>
+        /// Updates all packages for the project in the current working directory.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <example>
+        /// <code>
+        /// <![CDATA[
+        ///     NpmUpdate();
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Update")]
+        public static void NpmUpdate(this ICakeContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+            
+            context.NpmUpdate(new NpmUpdateSettings());
+        }
+
+        /// <summary>
+        /// Updates packages using the speUpdatefied settings.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">The settings.</param>
+        /// <example>
+        /// <para>Use speUpdatefic log level ('npm Update')</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var settings = new NpmUpdateSettings();
```

Comment:
Would be good to do something with the settings (e.g. set `global` flag)

Reply:
added example